resources/js/components/common/DataLoading.vue

```diff
@@ -80,7 +80,11 @@
                 if (error && error.code === "ERR_CANCELED") {
                     this.noResults = false;
                     this.error = true;
-                }  else {
+                }
+                if (error && error.response?.data?.message === "The item should not have associated processes") {
```

Comment:
hi @mcraeteisha , could we use a more generic condition? This is because DataLoading.vue is used in multiple places and a more generic approach could help solve the problem in other contexts... for instance:
```
if (error && error.code === "ERR_BAD_REQUEST") {
```

Reply:
Good feedback! Thanks, Eleazar :)